service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java

```diff
@@ -84,14 +85,22 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         Placement placement = mapper.convertValue(caseDetails.getData().get("placement"), Placement.class)
             .setChild(child);
 
-        caseProperties.put("placements", placementService.setPlacement(caseData, placement));
+        caseProperties.put("confidentialPlacements", placementService.setPlacement(placement, caseData.getConfidentialPlacements()));
```

Comment:
placement should be set only once

Reply:
Replied to you on slack, I am going to address this in separate PR if you are ok with that?